src/transformState.js

```diff
@@ -5,7 +5,25 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const command of actions) {
```

Comment:
```suggestion
  for (const action of actions) {
```

Reply:
![image](https://user-images.githubusercontent.com/99331793/189646893-df0623a6-6371-4772-92fc-cdbc8f3016ca.png)
fix it